saleor/core/jwt.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime
+
+import graphene
+import jwt
+from django.conf import settings
+
+from ..account.models import User
+
+JWT_ALGORITHM = "HS256"
+JWT_AUTH_HEADER = "HTTP_AUTHORIZATION"
+JWT_AUTH_HEADER_PREFIX = "JWT"
+JWT_ACCESS_TYPE = "access"
+JWT_REFRESH_TYPE = "refresh"
+JWT_REFRESH_TOKEN_COOKIE_NAME = "refreshToken"
+
+
+def jwt_base_payload(exp_delta=None):
+    payload = {
+        "iat": datetime.utcnow(),
+    }
+    if exp_delta:
+        payload["exp"] = datetime.utcnow() + exp_delta
+    return payload
+
+
+def jwt_user_payload(user, token_type, exp_delta, additional_payload=None):
+    if settings.JWT_DONT_EXPIRE:
```

Comment:
I don't think so; some people have deployments working in production mode and we would break them by turning on expiration there, since our frontend apps don't support that yet.